docs/docs/concepts/fleets.md

```diff
@@ -275,6 +275,10 @@ Define a fleet configuration as a YAML file in your project directory. The file
 
     3.&nbsp;The user specified should have passwordless `sudo` access.
 
+    4.&nbsp;The SSH server should be running and configured with `AllowTcpForwarding yes` in `/etc/ssh/sshd_config`.
+
+    5.&nbsp;The firewall should allow SSH and forbid any other connections from external networks.
```

Comment:
I think we have some ambiguity in this whole block.
* Almost all “should” must actually be “must” (as defined in [RFC 2119](https://www.rfc-editor.org/rfc/rfc2119)).
* The last requirement is “The firewall _must_ allow SSH and _should_ forbid any other connections from external networks” (the second part is for security reasons, it's not a strict requirement, right?).

I would also add that, as most communication libraries use ports from the ephemeral range, in cluster fleets the firewall must allow the internal network connections in that range (the easiest way is to allow all internal connections).